### PR TITLE
Use argparse instead of getopt

### LINKED ISSUE #15: Use argparse instead getopts
Why is the antiquated getopts used in this package.

See this: https://www.python.org/dev/peps/pep-0389/#why-aren-t-getopt-and-optparse-enough

argparse has been the preferred way to do argument parsing for quite some time

### PR DESCRIPTION
**Pull request by IvanMalison**
Fixes #15

### CODE REVIEW

**Comment by jacobsalmela** [comment]
Sorry, I haven't checked on this for a while.  I'll try this out and let you know.

**Comment by jacobsalmela** [comment]
I think some of your indents are messed up.  Can you fix those?

**Comment by IvanMalison** [comment]
Where? This file uses 2 space indents, right.

File: tccutil.py [hunk 1]
```diff
@@ -75,6 +50,42 @@ def sudo_required():
 
 def open_database():
   #------------------------
+parser = argparse.ArgumentParser(description='Modify Accesibility Preferences')
```
Comment (jacobsalmela) [inline tccutil.py]: Looks like it's not indented starting here.

**Comment by jacobsalmela** [comment]
Looks like line 53

File: tccutil.py (hunk 1 above)
Comment (IvanMalison) [inline tccutil.py]: No, that is supposed to be at the top level.

Comment (IvanMalison) [inline tccutil.py]: its a global variable.

Comment (jacobsalmela) [inline tccutil.py]: Why is it under the `open_database` function?

Comment (jacobsalmela) [inline tccutil.py]: bump

Comment (IvanMalison) [inline tccutil.py]: Sorry about that. Tried to do some fancy hunk based staging stuff when I had another refactor in progress.

**Comment by jacobsalmela** [comment]
OK I'll test it out and merge if it's working as expected.  Thanks!

**Comment by jacobsalmela** [comment]
`--version` doesn't seem to print anything; I think it is missing from `main()`.  Could you also bump the `util_version` to 1.2.2.

**Comment by jacobsalmela** [comment]
@IvanMalison If you fix `--version` per above, I'll merge.

**Comment by IvanMalison** [comment]
okay let me take a look

**Pull request merged**